ux.symfony.com/src/Service/CommonMark/ConverterFactory.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Service\CommonMark;
+
+use League\CommonMark\CommonMarkConverter;
+use League\CommonMark\Extension\CommonMark\Node\Block\FencedCode;
+use League\CommonMark\Extension\ExternalLink\ExternalLinkExtension;
+use League\CommonMark\Extension\Mention\MentionExtension;
+use Symfony\Component\DependencyInjection\Attribute\AsDecorator;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+#[AsDecorator('twig.markdown.league_common_mark_converter_factory')]
+final class ConverterFactory
+{
+    public function __invoke(): CommonMarkConverter
+    {
+        $converter = new CommonMarkConverter([
+            'mentions' => [
+                'github_handle' => [
+                    'prefix' => '@',
+                    'pattern' => '[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}(?!\w)',
+                    'generator' => 'https://github.com/%s',
+                ],
+                'github_issue' => [
+                    'prefix' => '#',
+                    'pattern' => '\d+',
+                    'generator' => 'https://github.com/symfony/ux/issues/%d',
+                ],
+            ],
+        ]);
+
+        $converter->getEnvironment()
+            ->addRenderer(FencedCode::class, new CodeRenderer(), 10)
+            ->addExtension(new ExternalLinkExtension())
```

Comment:
Can we restrict a bit the URLs ?

Reply:
I believe so yes, what did you have in mind?

Reply:
No you're right let's wait _if_ this is needed :)